rosidl_typesupport_cpp/resource/srv__type_support.cpp.em

```diff
@@ -109,12 +120,80 @@ static const type_support_map_t _@(service.namespaced_type.name)_service_typesup
   &_@(service.namespaced_type.name)_service_typesupport_data.data[0],
 };
 
+
+
+#ifdef __cplusplus
```

Comment:
Is the `#ifdef` necessary? This is most certainly a cpp file, so maybe we can omit it.

Reply:
96e23a359ffef893646e99216af56499e852e3dd